internal/command/commands/serviceset.go

```diff
@@ -140,13 +143,99 @@ func (s *ServiceSet) Run(args []string) int {
 		if *envName == "" {
 			*envName = utils.FetchKey(ENV_NAME_KEY)
 		}
+		isEnvPresent := len(*envName) > 0
+		isFilePresent := len(*setFilePath) > 0
+		isServiceSetNamePresent := len(*serviceSetName) > 0
+
+		if !isEnvPresent {
+			s.Logger.Error("--env is mandatory")
+			return 1
+		}
+		if isFilePresent && isServiceSetNamePresent {
+			s.Logger.Error("--name should not be provided when --setFile is provided.")
+			return 1
+		} else if !isFilePresent && !isServiceSetNamePresent {
+			s.Logger.Error("Please provide either --name or --setFile.")
+			return 1
+		}
+		emptyFileCreatedParameters := emptyParameters(map[string]string{"--env": *envName, "--setFile": *setFilePath})
+		if len(emptyFileCreatedParameters) == 0 {
+			// Create a service set in Temporary Table
+			configData, err := file.Read(*setFilePath)
+			if err != nil {
+				s.Logger.Error("Unable to read from " + *setFilePath + "\n" + err.Error())
+				return 1
+			}
+
+			var parsedConfig interface{}
+
+			if strings.Contains(*setFilePath, ".yaml") || strings.Contains(*setFilePath, ".yml") {
+				err = yaml.Unmarshal(configData, &parsedConfig)
+				if err != nil {
+					s.Logger.Error("Unable to parse YAML. " + err.Error())
+					return 1
+				}
+			} else if strings.Contains(*setFilePath, ".json") {
+				err = json.Unmarshal(configData, &parsedConfig)
+				if err != nil {
+					s.Logger.Error("Unable to parse JSON. " + err.Error())
+					return 1
+				}
+			} else {
+				s.Logger.Error("Unrecognized file format")
+				return 1
+			}
+			serviceDataMap := parsedConfig.(map[string]interface{})
+
+			serviceSetClient.CreateTempServiceSet(parsedConfig)
+			s.Logger.Success(fmt.Sprintf("Temporary ServiceSet: %s initialised Successfully.", serviceDataMap["name"].(string)))
+
+			var forceDeployServices []serviceset.ListEnvService
+			if !*force {
+				//get list of env services
+				s.Logger.Debug(fmt.Sprintf("Env Services of service-set %s and env %s", *serviceSetName, *envName))
+				serviceSetList, err := serviceSetClient.ListEnvServices(fmt.Sprintf("%v", serviceDataMap["name"]), *envName, "conflictedVersion", true)
```

Comment:
See if we can create a method from this as similar code is present in deploy with name